azurelinuxagent/common/protocol/wire.py

```diff
@@ -908,49 +918,59 @@ def _call_hostplugin_with_container_check(self, host_func):
                              log_event=True)
                 raise
 
+    def __send_request_using_host_channel(self, host_func):
+        """
+        Calls the host_func on host channel with retries for stale goal state and handles any exceptions, consistent with the caller for direct channel.
+        """
```

Comment:
Can you also add the possible return values here too? Just to make it a bit more clearer without having to go in the internal functions

Reply:
Sure, that's a good point.